cpp/modules/deck.gl/core/src/viewports/viewport.h

```diff
@@ -90,29 +90,25 @@ class Viewport {
   auto metersPerPixel() -> double;
   auto projectionMode() -> PROJECTION_MODE;
 
-  /**
-   * Projects xyz (possibly latitude and longitude) to pixel coordinates in
-   * window using viewport projection parameters
-   * - [longitude, latitude] to [x, y]
-   * - [longitude, latitude, Z] => [x, y, z]
-   * Note: By default, returns top-left coordinates for canvas/SVG type render
-   *
-   * @param lngLat - [lng, lat] or [lng, lat, Z]
-   * @param topLeft - Whether projected coords are top left
-   * @return {Array} - [x, y] or [x, y, z] in top left coords
-   */
+  /// \brief Projects xyz (possibly latitude and longitude) to pixel coordinates in
+  /// \brief window using viewport projection parameters
+  /// - [longitude, latitude] to [x, y]
+  /// - [longitude, latitude, Z] => [x, y, z]
+  /// \note By default, returns top-left coordinates for canvas/SVG type render
+  ///
+  /// \param lngLat [lng, lat] or [lng, lat, Z]
+  /// \param topLeft Whether projected coords are top left
+  /// \return {Array} - [x, y] or [x, y, z] in top left coords
```

Comment:
We should get rid of return types (types in general) from our docs, while useful in JS, they are not needed in C++